spec/rails_helper.rb

```diff
@@ -132,9 +132,8 @@
     Affiliate.OTHER
 
     # initialize price groups
-    @nupg = PriceGroup.find_or_create_by(name: Settings.price_group.name.base, is_internal: true, display_order: 1)
-    @nupg.save(validate: false)
-    PriceGroup.find_or_create_by(name: Settings.price_group.name.external, is_internal: false, display_order: 3).save(validate: false)
+    PriceGroup.setup_global(name: Settings.price_group.name.base, is_internal: true)
```

Comment:
I think this will fix the specs:
```suggestion
    @nupg = PriceGroup.setup_global(name: Settings.price_group.name.base, is_internal: true)
```